interop/src/clients/InteropClient/InteropClient.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,390 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 77;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		16029C932D37200E007B20A0 /* WireCoreCrypto.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 16029C902D371F0D007B20A0 /* WireCoreCrypto.framework */; };
+		16029C942D37200E007B20A0 /* WireCoreCrypto.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 16029C902D371F0D007B20A0 /* WireCoreCrypto.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+/* End PBXBuildFile section */
+
+/* Begin PBXCopyFilesBuildPhase section */
+		16029C952D37200E007B20A0 /* Embed Frameworks */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "";
+			dstSubfolderSpec = 10;
+			files = (
+				16029C942D37200E007B20A0 /* WireCoreCrypto.framework in Embed Frameworks */,
+			);
+			name = "Embed Frameworks";
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXCopyFilesBuildPhase section */
+
+/* Begin PBXFileReference section */
+		16029C902D371F0D007B20A0 /* WireCoreCrypto.framework */ = {isa = PBXFileReference; explicitFileType = wrapper.framework; path = WireCoreCrypto.framework; sourceTree = BUILT_PRODUCTS_DIR; };
+		166BD2BC2D28123F00E9FABB /* InteropClient.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = InteropClient.app; sourceTree = BUILT_PRODUCTS_DIR; };
+		166BD2E12D30296B00E9FABB /* WireCoreCrypto.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = WireCoreCrypto.xcodeproj; path = "../../../../crypto-ffi/bindings/swift/WireCoreCrypto/WireCoreCrypto.xcodeproj"; sourceTree = SOURCE_ROOT; };
+/* End PBXFileReference section */
+
```

Comment:
It should be possible to script creation of these auto-generated project files, right?
Why not use something like https://github.com/yonaskolb/XcodeGen and avoid checking in auto-generated files into the repo?

Reply:
Yep, that would probably work but that's another dependency on a third party tool. In this case I'm not sure if it's worth since I suspect we'll rarely change the project files.

I'm hoping soon it will be supported to build iOS app using the swift package manager.